db/migrate/20221005121427_add_seats_to_reservation.rb

```diff
@@ -0,0 +1,5 @@
+class AddSeatsToReservation < ActiveRecord::Migration[7.0]
+  def change
+    add_column :reservations, :seats, :integer, array: true
```

Comment:
why do you need seats in a reservation when you have them in related tickets? 🤔

Reply:
it really helps me manipulate them in views and controller. I know this is not a perfect solution but it works fine and allows me to move one to other things with what time is left.